### ISSUE TITLE
[BUG] WP CLI module evaluates path during build

### ISSUE THREAD

**Issue by TimothyBJacobs** [lucatume/wp-browser#294]
**Environment**
OS: Mac
PHP version: 7.3
Installed Codeception version: v3.1.0
Installed wp-browser version: 2.2.19
WordPress version: 5.2
Local development environment: Local
WordPress structure and management: Default

**Codeception configuration file**  
```yaml
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
    commands:
        - Codeception\Command\GenerateWPUnit
        - Codeception\Command\GenerateWPRestApi
        - Codeception\Command\GenerateWPRestController
        - Codeception\Command\GenerateWPRestPostTypeController
        - Codeception\Command\GenerateWPAjax
        - Codeception\Command\GenerateWPCanonical
        - Codeception\Command\GenerateWPXMLRPC
params:
    - tests/codeception.env

settings:
    shuffle: true
```

**Suite configuration file**
```yaml
actor: AcceptanceTester
modules:
    enabled:
        - WPDb
        - WPBrowser
        - WPCLI
        - \Helper\Acceptance
    config:
        WPDb:
            dsn: 'mysql:host=%TEST_SITE_DB_HOST%;port=%TEST_SITE_DB_PORT%;dbname=%TEST_SITE_DB_NAME%'
            user: '%TEST_SITE_DB_USER%'
            password: '%TEST_SITE_DB_PASSWORD%'
            dump: 'tests/_data/dump.sql'
            #import the dump before the tests; this means the test site database will be repopulated before the tests.
            populate: true
            # re-import the dump between tests; this means the test site database will be repopulated between the tests.
            cleanup: true
            waitlock: 10
            url: '%TEST_SITE_WP_URL%'
            urlReplacement: true #replace the hardcoded dump URL with the one above
            tablePrefix: '%TEST_SITE_TABLE_PREFIX%'
        WPBrowser:
            url: '%TEST_SITE_WP_URL%'
            adminUsername: '%TEST_SITE_ADMIN_USERNAME%'
            adminPassword: '%TEST_SITE_ADMIN_PASSWORD%'
            adminPath: '%TEST_SITE_WP_ADMIN_PATH%'
        WPCLI:
            path: '/bitnami/wordpress'
```

**Describe the bug**
I run acceptance tests within a docker container using `docker exec`. When I try to build my tests on the host machine, the WPCLI module fails.

**Output**
```
 WPCLI module is not configured!

  Specified path [/bitnami/wordpress] is not a directory
```

**To Reproduce**
Create a docker image where the root directory of WordPress is a directory that does not exist on your host machine.

**Expected behavior**
For the WP Cli directory not to be evaluated during build time.

**Additional context**
A similar problem occurs when using `ssh` with the WPCLI module. 

The `WPFilesystem` module doesn't appear to validate the `wpRootPath` and WP-CLI will error out at runtime if the `path` isn't correct, so perhaps the validation can be safely dropped?

**Comment by lucatume** [comment]
@TimothyBJacobs thanks for opening an issue and taking the time to fill in all the details.

I think your proposal to move the `path` check from build time to run time makes a lot of sense and I'm addressing it in the PR #298 .

Being there I've pulled over some WPCLI module refactorings I've done while working on SQLite integration, those should not affect the issues, but could provide context into the, otherwise, "hefty" PR.

**Comment by TimothyBJacobs** [comment]
👍 thanks! Happy to test whenever it's ready.

**Comment by lucatume** [comment]
Fixed in #298
